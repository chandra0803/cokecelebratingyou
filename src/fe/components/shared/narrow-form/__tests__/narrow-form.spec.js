import React from 'react';
import NarrowForm from '../narrow-form';
import renderer from 'react-test-renderer';

test( 'Form renders', () => {
    const props = {
        login: {
            toggleModal: true,
        },
        content: {
            'system.general.help_text': '???system.general.HELP_TEXT???',
            'system.general.help_title': '???system.general.HELP_TITLE???'
        },
        toggleModalDisplay: function( toggleModal ) {
            return !toggleModal;
        }
    };
    const component = renderer.create(
        <div>
            <NarrowForm { ...props }/>
        </div>
    );

    const tree = component.toJSON();
    expect( tree ).toMatchSnapshot();
} );
