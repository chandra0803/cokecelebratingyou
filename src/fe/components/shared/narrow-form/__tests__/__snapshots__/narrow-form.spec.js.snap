// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders 1`] = `
<div>
  <div
    className="narrow-form-page undefined"
  >
    <form
      action={undefined}
      className="narrow-form"
      method={undefined}
    />
    <div
      className="theme-image"
    />
  </div>
</div>
`;
